tools/remote_build/README.md

```diff
@@ -17,22 +17,27 @@ and tests run by Kokoro CI.
 
 ## Running remote build manually from dev workstation
 
+*At the time being, tools/bazel.sh is used instead of invoking "bazel" directly
```

Comment:
Would it be a good idea to hoist this somewhere higher? Perhaps the top-level README?

Reply:
This is mostly meant as a heads up for the team (others can't run the manual remote bazel runs anyway) so we'd announce internally.